src/value/mod.rs

```diff
@@ -258,6 +260,21 @@ pub trait Element {
     /// This will return `None` if the type is not `int` or the value is any `null`.
     fn as_any_int(&self) -> Option<&AnyInt>;
 
+    /// Returns a reference to the underlying float value for this element.
+    ///
+    /// This will return `None` if the type is not `float` or the value is any `null`.
+    fn as_float(&self) -> Option<&f64>;
```

Comment:
This should just return `Option<f64>` as `f64` is a cheap value.  The reason why `AnyInt` gets borrowed is due to it not supporting `Copy` trait (because of the `BigInt` variant).